packages/safety-suite-api/src/api/eForms.ts

```diff
@@ -0,0 +1,181 @@
+  driverId: number;
+  createdByUserId: number;
+  creationDate: string;
+}
+
+export interface CreateTemplateUrlRequest {
+  file: Blob;
+  title: string;
+}
+
+export interface CreateFormTemplateRequest {
+  hellosignId: string;
+  title: string;
+}
+
+export interface CreateSignatureRequestRequest {
+  hellosignTemplateIds: [string];
+  driverId: number;
+}
+
+export interface GetSignaturesQuery {
+  statusCode: string;
+}
+
+export interface GetFormTemplateByIdQuery {
+  formTemplateId: string;
+}
+
+export interface GetSignatureFileQuery {
+  hellosignTemplateId: string;
+}
+
+// HelloSign
+export function createTemplateUrl(
+  request: CreateTemplateUrlRequest,
+  apiOptions: ApiOptions = {}
+): Request<ApiResponse<HelloSignFormTemplate>> {
+  return runApi({
+    method: 'POST',
+    urlRoot: 'eFormsUrlRoot',
+    route: '/api/1.0/vendors/hellosign/createTemplateUrl',
+    apiOptions,
+    requestOptions: {body: request}
+  });
+}
+
+export function getTemplateEditUrl(
+  formTemplateId: number,
+  apiOptions: ApiOptions = {}
+): Request<ApiResponse<SignatureRequestUrl>> {
+  return runApi({
+    method: 'GET',
+    urlRoot: 'eFormsUrlRoot',
+    route: `/api/1.0/vendors/hellosign/editTemplateUrl/${formTemplateId}`,
+    apiOptions
+  });
+}
+
+// Form Templates
+export function getFormTemplates(
+  apiOptions: ApiOptions = {}
+): Request<ApiResponse<FormTemplate[]>> {
+  return runApi({
+    method: 'GET',
+    urlRoot: 'eFormsUrlRoot',
+    route: '/api/1.0/formTemplates',
+    apiOptions
+  });
+}
+
+export function getFormTemplateById(
+  query: GetFormTemplateByIdQuery,
+  apiOptions: ApiOptions = {}
+): Request<ApiResponse<FormTemplate>> {
+  return runApi({
+    method: 'GET',
+    urlRoot: 'eFormsUrlRoot',
+    route: '/api/1.0/formTemplates',
+    apiOptions,
+    requestOptions: {query}
+  });
+}
+
+export function createFormTemplates(
+  request: CreateFormTemplateRequest,
+  apiOptions: ApiOptions = {}
+): Request<ApiResponse<FormTemplate>> {
+  return runApi({
+    method: 'POST',
+    urlRoot: 'eFormsUrlRoot',
+    route: '/api/1.0/formTemplates',
+    apiOptions,
+    requestOptions: {body: request}
+  });
+}
+
+// Signature Requests
+export function getSignatures(
+  query: GetSignaturesQuery,
+  apiOptions: ApiOptions = {}
+): Request<ApiResponse<SignatureRequest[]>> {
+  return runApi({
+    method: 'GET',
+    urlRoot: 'eFormsUrlRoot',
+    route: '/api/1.0/signatures',
+    apiOptions,
+    requestOptions: {query}
+  });
+}
+
+export function createSignature(
+  request: CreateSignatureRequestRequest,
+  apiOptions: ApiOptions = {}
+): Request<ApiResponse<SignatureRequest>> {
+  return runApi({
+    method: 'POST',
+    urlRoot: 'eFormsUrlRoot',
+    route: '/api/1.0/signatures',
+    apiOptions,
+    requestOptions: {body: request}
+  });
+}
+
+export function getSignatureFile(
+  signatureId: number,
+  query: GetSignatureFileQuery,
+  apiOptions: ApiOptions = {}
+): Request<ApiResponse<Blob>> {
```

Comment:
No need for `ApiResponse` here, and you need `responseType: 'blob'` in the `requestOptions`